src/components/usa-breadcrumb/usa-breadcrumb.stories.js

```diff
@@ -0,0 +1,24 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-breadcrumb.twig";
+import Data from "./usa-breadcrumb.json";
+import WrapData from "./usa-breadcrumb--wrap.json";
+
+export default {
+  title: "Components/Breadcrumb",
+  argTypes: {
+    modifier: {
+      control: "text",
+    },
+    aria_label: {
+      control: "text",
+    },
+  },
+};
```

Comment:
I believe we don't need to declare the control type for these - the JSON should get us there